frontstage/views/register.py

```diff
@@ -256,14 +249,14 @@ def register_enter_your_details():
 
         except JWTError:
             # TODO Provide proper logging
-            logger.warning('JWT scope could not be validated.')
-            return abort(500, '{"message":"There was a problem with the Authentication service please contact a member of the ONS staff"}')
+            logger.warning('"event" : "JWT scope could not be validated."')
+            return abort(500, '{"event" : "There was a problem with the Authentication service please contact a member of the ONS staff"}')
 
         except MissingTokenError as e:
-            logger.warning("Missing token error, error is: {}".format(e))
-            logger.warning("Failed validation")
+            logger.warning('"event" : "missing token", "Missing token, error is" : "{}"'.format(e))
```

Comment:
logger.warning('"event" : "missing token", "Missing token, error is" : "{}"'.format(e))
this doesn't follow the key value pair pattern